www/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "types": ["@astrojs/image/client"],
     // Skip type checking of external libraries.
     "skipLibCheck": true,
-    "importsNotUsedAsValues": "preserve" // fixed through linting
```

Comment:
We need this for Astro not to complain i think

Reply:
@juliusmarminge 

`importsNotUsedAsValues` is deprecated in favour of `verbatimModuleSyntax`

https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#verbatimmodulesyntax

I am not 100% but it looks like we are waiting for type-fest release with TS 5.0 compatibility: https://github.com/sindresorhus/type-fest/pull/573 (PR merged, but not released yet).

Reply:
I see! Let's hope that solves it once that's released